core/src/main/java/nl/nn/adapterframework/pipes/ChecksumPipe.java

```diff
@@ -54,8 +54,23 @@ public class ChecksumPipe extends StreamingPipe {
 	public enum ChecksumType {
 		MD5,
 		SHA,
+		SHA256("SHA-256"),
+		SHA512("SHA-512"),
 		CRC32,
-		ADLER32
+		ADLER32;
```

Comment:
stiekem moet de deze ook testen

Reply:
er is op regel 120: 	public void testFileChecksumADLER32() throws Exception